pkg/ttl/cache/table.go

```diff
@@ -185,28 +183,40 @@ func (t *PhysicalTable) ValidateKeyPrefix(key []types.Datum) error {
 	return nil
 }
 
-// EvalExpireTime returns the expired time
+// EvalExpireTime returns the expired time.
+// It uses the global timezone to compute the expired time.
+// Then we'll reset the returned expired time to the same timezone as the input `now`.
```

Comment:
In which case can this be executed in a different time zone than the global time zone?

Reply:
For this PR, the session time zone will be always different with global time zone if the global time zone is not `UTC`. That is because we set the session time zone to `UTC` to avoid some problems. For example, the DST problem that may case a wrong inserted timestamp value.